### PR TITLE
Restore old flake8

### PR DESCRIPTION
**Pull request by gberaudo**
The new flake8 is breaking all our original images.
The flake8 update should be part of a new tag.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ cee_syslog_handler==0.4.1
 codacy-coverage==1.3.7
 cornice==3.0.0
 coverage==4.4.1
-flake8==3.5.0
+flake8==3.4.1
```
Comment (pvalsecc) [inline requirements.txt]: Please append `# rq.filter: <3.5.0`. Otherwise, requires.io will upgrade it back to 3.5.

**Comment by sbrunner** [comment]
Why that?

**Comment by gberaudo** [comment]
Because it is an unnecessary breaking change.
Flake8 3.5.0 will be part of release_1, see https://github.com/camptocamp/c2cwsgiutils/tree/switch_to_stretch.

**Comment by sbrunner** [comment]
OK :-)

**Pull request merged**